R/SS_readctl_3.30.R

```diff
@@ -1767,8 +1766,7 @@ translate_3.30_to_3.24_var_adjust <- function(Variance_adjustment_list = NULL,
 #' @param Q_options The Q options list element in the 3.30
 #'  control file r4ss list output generated from [SS_readctl].
 #' @param Nfleets Number of fleets in the model
-#' @param fleetnames Name of the fleets. Defaults to fleet numbers, in the order
-#' @return A dataframe containing the 3.24 Q setup.
```

Comment:
I think line 1771 was removed by mistake?

Reply:
Yep, good catch! I'll push up a commit to add it back in.